src/reducers/filter.js

```diff
@@ -0,0 +1,12 @@
+import { FILTER_BOOK } from '../actions';
+
+const filterCategory = (state = '', action) => {
+  switch (action.type) {
+    case FILTER_BOOK:
+      return action.category;
+    default:
+      return state;
+  }
+};
+
```

Comment:
it should default to `"all"`according to [documentation](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/ItDefaultsToAll.png) not empty string